app/+QSP/@CohortGeneration/CohortGeneration.m

```diff
@@ -41,7 +34,7 @@
         RefParamName = '' % Parameters.Name
         GroupName = ''
         Method = 'Distribution' 
-        SaveInvalid = 'Save all virtual subjects' % 'all' = 'Save all virtual subjects' or 'valid' = 'Save valid virtual subjects'
+        SaveInvalid = 'all' % = 'Save all virtual subjects' or 'valid' = 'Save valid virtual subjects'
```

Comment:
I think ultimately we wanted to use the full strings in the UI. It doesn't matter if that is represented as a shortened string in the backend.